pkg/storage/s3/s3.go

```diff
@@ -88,7 +89,19 @@ func (d *driver) ConfigEnv() (envs []corev1.EnvVar, err error) {
 		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_BUCKET", Value: d.Config.Bucket},
 		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_REGION", Value: d.Config.Region},
 		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_REGIONENDPOINT", Value: d.Config.RegionEndpoint},
-		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_ENCRYPT", Value: fmt.Sprintf("%v", d.Config.Encrypt)},
+		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_SECURE", Value: strconv.FormatBool(d.Config.Secure)},
+		corev1.EnvVar{Name: "REGISTRY_STORAGE_S3_ENCRYPT", Value: strconv.FormatBool(d.Config.Encrypt)},
+		corev1.EnvVar{
```

Comment:
I'm not 100% sure (according to the documentation) whether or not the KMS KeyID is sensitive data or not, so maybe it doesn't need to be stored in a secret ...
https://docs.docker.com/registry/storage-drivers/s3/

Reply:
I confirmed that the KMS Key ID is not sensitive information, so refactoring to just use the field